FrEIA/modules/invertible_resnet.py

```diff
@@ -134,20 +133,22 @@ def lipschitz_correction(self):
                     self.layers[i].weight.data *= self.spectral_norm_max / spectral_norm
 
 
-    def forward(self, x, c=[], rev=False):
+    def forward(self, x, c=[], rev=False, jac=True):
+        jac = self._jacobian(x, c, rev=rev)
```

Comment:
This is costly, so put it a check for `jac == True`?

Reply:
Good point, I will do that! :)